jekyll/_cci2/jenkins-converter.md

```diff
@@ -5,6 +5,7 @@ short-title: "Jenkins Converter Introduction"
 description: "Starting point for how to use the Jenkins Converter"
 categories: [getting-started]
 order: 1
+noindex: true
```

Comment:
can we add a `noindex: false` by default in the config file? This will help to make sure that every page is always indexable unless they have `noindex: true`

Reply:
Yeah, would this go under the overall site settings?
```
# Site settings
title: "CircleCI Documentation"
email: docs@cirleci.com
description: > # this means to ignore newlines until "baseurl:"
  The documentation site for CircleCI, the fastest testing platform on the
  Internet.
```
Or would it go under the specific collection(s)?
```
  cci2:
    name: "2.0 Docs"
    site_section: true
    output: true
    permalink: /2.0/:path/
    prefix: "/2.0"
    selectorName: "2.0"
```

Reply:
@devinmarieb overall site settings please since the layout is used a little bit everywhere.